fft_simulation/analytical_cases.py

```diff
@@ -114,17 +209,21 @@ def main(geometry_type, buffer):
 
         # plot sections of the chi distribution
         fig, axes = plt.subplots(2, 3, figsize=(10, 5), dpi=120)
```

Comment:
A plot () function should be integrated for each class instead of having an if statement with two very similar functions. You could even create a PArent class to both where you define the plot function.

Reply:
I created a parent class for plotting